channeldb/channel.go

```diff
@@ -3783,6 +3786,12 @@ type RevocationLog struct {
 	// channel directly spendable by the remote node.
 	RemoteBalance lnwire.MilliSatoshi
 
+	// LocalOutputIndex specifies our output index in this commitment.
+	LocalOutputIndex uint32
```

Comment:
Is this actually necessary given that we know the balances? As we use a modified BIP 69 here, we know the first sorting will be by the amounts, so if we know even just our amount, then we can derive their output index. 

Alternatively, if they're breaching, then we can distinguish our output, as it'll be a p2wkh instead of a p2wsh.